src/Util/Root.cpp

```diff
@@ -25,25 +26,39 @@ void Root::Update() {
     struct StackInfo {
         std::shared_ptr<GameObject> m_GameObject;
         Transform m_ParentTransform;
+        bool operator()(const StackInfo& lhs, const StackInfo& rhs) const {
+            return  lhs.m_GameObject->GetZIndex()> rhs.m_GameObject->GetZIndex();
+        }
```

Comment:
what is this for?

Reply:
wait this is `operator()`???

Reply:
![SUSSY](https://cdn.7tv.app/emote/60ccf4479f5edeff9938fa77/1x.webp)